### PR TITLE
upadate gulp.js file

### PR DESCRIPTION
**Pull request by aalikaram**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -11,18 +11,22 @@ USWDS SASS GULPFILE
...
 const postcss = require("gulp-postcss");
 const replace = require("gulp-replace");
 const sass = require("gulp-sass");
 const sourcemaps = require("gulp-sourcemaps");
-const uswds = require("./node_modules/uswds-gulp/config/uswds");
+const uswds = require("./node_modules/uswds-gulp/config/uswds");*/
+
+
+
+/*sass.compiler = require("sass");*/
```
Comment (mejiaj) [inline gulpfile.js:26]: Please delete commented code from line 14-26.

File: gulpfile.js [hunk 2]
```diff
@@ -133,22 +130,42 @@ gulp.task(
...
+/*gulp.task('update', gulp.series(
   'copy-uswds-core',
   'copy-uswds-lib',
   'copy-uswds-settings',
   'copy-uswds-fonts',
   'copy-uswds-images',
   'copy-uswds-js',
   'build-sass',
-));
+));*/
```
Comment (mejiaj) [inline gulpfile.js:143]: Delete commented code.

File: gulpfile.js [hunk 3]
```diff
@@ -133,22 +130,42 @@ gulp.task(
...
 
-gulp.task("watch-sass", function() {
-  gulp.watch(`${PROJECT_SASS_SRC}/**/*.scss`, gulp.series("build-sass"));
-});
 
-gulp.task("watch", gulp.series("build-sass", "watch-sass"));
+exports.init =gulp.series(
```
Comment (mejiaj) [inline gulpfile.js:146]:
```suggestion
exports.init = gulp.series(
```

Needs a space after the `=`.

File: assets/stylesheets/lib/uswds-sass/_uswds-theme-color.scss [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+$theme-text-reverse-color: "white";
+
+// Links
+$theme-link-color: "primary";
+$theme-link-visited-color: "violet-70v";
+$theme-link-hover-color: "primary-dark";
+$theme-link-active-color: "primary-darker";
+$theme-link-reverse-color: "base-lighter";
+$theme-link-reverse-hover-color: "base-lightest";
+$theme-link-reverse-active-color: "white";
```
Comment (mejiaj) [inline assets/stylesheets/lib/uswds-sass/_uswds-theme-color.scss:147]: Git shows these theme settings files as newly added files:

```bash
assets/stylesheets/lib/uswds-sass/_uswds-theme-color.scss
assets/stylesheets/lib/uswds-sass/_uswds-theme-components.scss
assets/stylesheets/lib/uswds-sass/_uswds-theme-custom-styles.scss
assets/stylesheets/lib/uswds-sass/_uswds-theme-general.scss
assets/stylesheets/lib/uswds-sass/_uswds-theme-spacing.scss
assets/stylesheets/lib/uswds-sass/_uswds-theme-typography.scss
assets/stylesheets/lib/uswds-sass/_uswds-theme-utilities.scss
assets/stylesheets/lib/uswds-sass/styles.scss
```

Where are these coming from? They shouldn't be there.

**Review by mejiaj — CHANGES_REQUESTED** [review]
Good start, added some comments based on testing.

Please create a `.nvmrc` file and append the latest LTS version of node to it. Otherwise you get an error with uswds-compile since node v14 is set on this project.

image

When running I see a lot of these warnings 
```bash
WARNING: `600px` is not a valid USWDS project breakpoint. Valid values: card, card-lg, mobile, mobile-lg, tablet, tablet-lg, desktop, desktop-lg, widescreen
         on line 13:5 of assets/stylesheets/lib/uswds-sass/core/mixins/_at-media.scss, in mixin `at-media`
         from line 85:12 of assets/stylesheets/site/components/_custom.scss
         from line 32:9 of main.scss
```
It could be fixed once we import the files to the right places.

There's also these:
```bash
^[[F[2022-03-22 08:59:26] ERROR `/assets/stylesheets/lib/img/usa-icons/expand_more.svg' not found.
[2022-03-22 08:59:26] ERROR `/assets/stylesheets/lib/fonts/source-sans-pro/sourcesanspro-regular-webfont.woff2' not found.
[2022-03-22 08:59:26] ERROR `/assets/stylesheets/lib/fonts/source-sans-pro/sourcesanspro-regular-webfont.woff' not found.
[2022-03-22 08:59:26] ERROR `/assets/stylesheets/lib/fonts/source-sans-pro/sourcesanspro-bold-webfont.woff2' not found.
[2022-03-22 08:59:26] ERROR `/assets/stylesheets/lib/fonts/source-sans-pro/sourcesanspro-regular-webfont.ttf' not found.
[2022-03-22 08:59:26] ERROR `/assets/stylesheets/lib/fonts/source-sans-pro/sourcesanspro-bold-webfont.woff' not found.
[2022-03-22 08:59:26] ERROR `/assets/stylesheets/lib/fonts/source-sans-pro/sourcesanspro-bold-webfont.ttf' not found.
```

File: gulpfile.js (hunk 1 above)
Comment (aalikaram) [reply gulpfile.js:26]: Done

File: gulpfile.js (hunk 2 above)
Comment (aalikaram) [reply gulpfile.js:143]: Done

File: assets/stylesheets/lib/uswds-sass/_uswds-theme-color.scss (hunk 4 above)
Comment (aalikaram) [reply assets/stylesheets/lib/uswds-sass/_uswds-theme-color.scss:147]: I didn't add them, it seems it been added when I was running the tasks

**Comment by mejiaj** [comment]
@aalikaram the structure isn't what's currently on `main` branch.

## Missing assets
We want to get a similar setup to the `main` screenshot on the left. Please update the tasks and run them so that they output the assets we need.
image

## Duplicate tasks
The `gulpfile.js` also has duplicate `init` and `update` tasks. Remove the tasks on line `98 and 99` and make sure your tasks work.

image

## Update dependencies

`package.json` still has the dependencies from `uswds-gulp`. We need to remove those npm dependencies listed [here](https://github.com/uswds/uswds-gulp#requirements)

You'll need to install the following packages via npm:

- autoprefixer
- del
- gulp@^4.0.2
- gulp-replace
- sass
- gulp-sass
- gulp-sourcemaps
- postcss
- gulp-postcss
- gulp-rename
- gulp-svg-sprite
- postcss-csso
- uswds@latest
- uswds-gulp@github:uswds/uswds-gulp

---

Overall I should expect the same directory structure as on `main`. Gulp tasks should be from `uswds-compile` **except** `init` and `update`.

**Pull request closed without merging**